### ISSUE TITLE
Project overview issue

### ISSUE THREAD

**Issue by tombackert** [tombackert/championship#2]
# Project Championship

### Goal
- Predict the upcoming Premier League champion

### Idea
- I'm no data scientist, neither a football analyst (eventhough I know a fair amount). So why not let a neural network figure out the correlations in the data that lead a team to the championship? I was good at building predicter systems in the past, so let's build a predicter to predict the upcoming champion

### Milestones
- Market analysis
- Decision on data
- Decision on model type
- Decision on mpv strat
- MPV development
- Make it a product!
- Make it better! (Delete, optimize, accelerate, automate)
- Scale it!

**Comment by tombackert** [comment]
### Data
- Match outcomes (wins, losses, goals)
- Player related (injuries, transfers)
- Team dynamics (home vs away performance)
- Weather data on game day

### Data challenges
- API integration (football-data.org)
- Web scraping (transfermarkt)
- Handling inconsistencies

**Comment by tombackert** [comment]
### Approaches (models)
- Regression models (poisson regression)
- Ensemble methodes (Random Forest, XGBoost)
- Time-series models (LSTM)
- Bayesian networks (probabilistic outcomes)
- Deep Learning (enough data?)
- Monte Carlo methodes

**Comment by tombackert** [comment]
### Deployment
- Simple dashboard
- Integration of live data (API setup)

**Comment by tombackert** [comment]
### Psychology of football
- Problem: Games can be decided by a teams state of mind 
- Football outcomes are highly linked to emotional aspects
- How can this be linked to external statistics?
- Where are the corelations between game outcomes and a teams state of mind?

**Comment by tombackert** [comment]
### Milestones
- Market analysis
  - How was this done before?
  - How can it be made better?
- Decision on data
- Decision on model type
- Decision on mpv strat
- MPV Development
  - Full ml pipeline
    - Data collection
    - EDA
    - Feature Engineering
    - Model selection/building/training
    - Evaluation (Metrics: Accuracy, Log loss, Brier score, comparision vs bookmaker models)
    - Deployment
  - Explaining model decision (XAI: SHAP, LIME)
- Reiteration
- Make it a product!
- Make it better
  - Updating model as new data comes in
  - Retraining pipeline
- Scale it!
